edward/data.py

```diff
@@ -34,20 +34,20 @@ def __init__(self, data):
         self.N = data.shape[0]
         self.counter = 0
 
-    def next(self, n_data=None):
+    def next(self, n_minibatch=None):
```

Comment:
Maybe in the same way that the distribution objects sample with a number given by `size`, `data.next()` should use have its argument be called `size`. My guess is that the distinction is that `size` is used generically for any context, and `n_minibatch`/`n_samples` is used specifically in the context of inference and criticism.